lib/airtable/entity/record.rb

```diff
@@ -0,0 +1,94 @@
+require 'time'
+# Main object for store Airtable Record entity
+module Airtable
+  module Entity
+    # Airtable Record entity
+    class Record
+      extend Forwardable
+      attr_reader :id, :created_at, :fields
+
+      def_delegators :@fields, :[], :[]=
+
+      def initialize(id, options = {})
+        @id = id
+        parse_options(options)
+      end
+
+      def new_record?
+        @id.nil? || @id.empty?
+      end
+
+      def __create__(base, name)
+        res = base.__make_request__(:post, name, fields: fields)
+        @id = res['id']
+        parse_options(fields: res['fields'], created_at: res['createdTime'])
+        self
+      end
+
+      def __update__(base, name)
+        args = [:patch, [name, @id].join('/'), fields: fields]
```

Comment:
Important: `name` (i.e. `tableName`) must be properly encoded as it may have special characters (e.g. https://github.com/Airtable/airtable-ruby/commit/3cc21eaf699be2eec2c0c135c042f61508cff878). Try it with a table that's named `Some &.#%?//  table`

Reply:
Done